src/components/Editor/Editor.js

```diff
@@ -584,6 +585,16 @@ class Editor extends React.Component {
                 />
               )}
             </label>
+            <label htmlFor="reading_time" className="Editor__reading_time">
+              <FormattedMessage
```

Comment:
You have to add translations here as well:
https://github.com/busyorg/busy/blob/new-design/src/locales/en.json